tensorflow/contrib/distributions/python/ops/bijectors/ordered.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Ordered bijector."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+from tensorflow.python.framework import tensor_shape
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import check_ops
+from tensorflow.python.ops import control_flow_ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.ops.distributions import bijector
+
+
+__all__ = [
+    "Ordered",
+]
+
+
+class Ordered(bijector.Bijector):
```

Comment:
Is there a more specific name we can use here? One worry I have is that Order statistics are also a valid bijector (and might be a thing in the future), and so the names can be confused.

Reply:
I would suggest call this the `Ordered` bijector as it is used similarly in Stan and PyMC3. The other variation of ordered statistics maybe better to call it a `Sorted` bijector?

Reply:
Yeah I guess that is my worry, which is Sorted and Ordered sound very similar (and I wouldn't trust myself to know which one to use :p). If we can make those sound different enough, make it easy for someone new to this stuff understand this bijector, and also make it easy for those using Stan / PyMC3 to know what this is, I'd be happy (I admit  this is an unfair question, in that I haven't provided any suggestions, but am currently thinking about a name, and will post back).

Reply:
It is not exactly the same thing as Ordered logistic (as in ordered logistic distribution ordered logistic regression) though - The ordered vector returned from this bijector is used to generate ordered logistic regression and ordered probit regression, so I think naming it Ordered bijector shouldnt be too confusing.